dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAIChatCompletion_AutoFunctionChoiceBehaviorTests.cs

```diff
@@ -165,46 +165,46 @@ public async Task SpecifiedInCodeInstructsConnectorToInvokeKernelFunctionAutomat
         Assert.Contains("GetCurrentDate", invokedFunctions);
     }
 
-    //[Fact]
-    //public async Task SpecifiedInPromptInstructsConnectorToInvokeKernelFunctionAutomaticallyForStreamingAsync()
-    //{
-    //    // Arrange
-    //    this._kernel.ImportPluginFromType<DateTimeUtils>();
-
-    //    var invokedFunctions = new List<string>();
-
-    //    this._autoFunctionInvocationFilter.RegisterFunctionInvocationHandler(async (context, next) =>
-    //    {
-    //        invokedFunctions.Add(context.Function.Name);
-    //        await next(context);
-    //    });
-
-    //    var promptTemplate = """"
-    //        template_format: semantic-kernel
-    //        template: How many days until Christmas?
-    //        execution_settings:
-    //          default:
-    //            temperature: 0.1
-    //            function_choice_behavior:
-    //              type: auto
-    //        """";
-
-    //    var promptFunction = KernelFunctionYaml.FromPromptYaml(promptTemplate);
-
-    //    string result = "";
-
-    //    // Act
-    //    await foreach (string c in promptFunction.InvokeStreamingAsync<string>(this._kernel))
-    //    {
-    //        result += c;
-    //    }
-
-    //    // Assert
-    //    Assert.NotNull(result);
-
-    //    Assert.Single(invokedFunctions);
-    //    Assert.Contains("GetCurrentDate", invokedFunctions);
-    //}
+    [Fact]
+    public async Task SpecifiedInPromptInstructsConnectorToInvokeKernelFunctionAutomaticallyForStreamingAsync()
+    {
+        // Arrange
+        this._kernel.ImportPluginFromType<DateTimeUtils>();
+
+        var invokedFunctions = new List<string>();
+
+        this._autoFunctionInvocationFilter.RegisterFunctionInvocationHandler(async (context, next) =>
+        {
+            invokedFunctions.Add(context.Function.Name);
+            await next(context);
+        });
+
+        var promptTemplate = """"
+            template_format: semantic-kernel
+            template: How many days until Christmas?
+            execution_settings:
+              default:
+                temperature: 0.1
+                function_choice_behavior:
+                  type: auto
+            """";
+
+        var promptFunction = KernelFunctionYaml.FromPromptYaml(promptTemplate);
+
+        string result = "";
+
+        // Act
+        await foreach (string c in promptFunction.InvokeStreamingAsync<string>(this._kernel))
+        {
+            result += c;
+        }
```

Comment:
Avoiding abbreviations.

```suggestion
        await foreach (string content in promptFunction.InvokeStreamingAsync<string>(this._kernel))
        {
            result += content;
        }
```